src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Double.cs

```diff
@@ -18,7 +18,8 @@ public sealed partial class Utf8JsonWriter
         /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
         /// </exception>
         /// <remarks>
-        /// Writes the <see cref="double"/> using the default <see cref="StandardFormat"/> (i.e. 'G').
+        /// Writes the <see cref="double"/> using a <see cref="StandardFormat"/> of 'G' on .NET Core
+        /// and 'G17' on .NET Framework.
```

Comment:
As with other cases, this wording is confusing, since based on the changes below it appears this will use G17 on, say, .NET Core 2.2.